Tests/test_imagecms.py

```diff
@@ -335,12 +335,26 @@ def test_extended_information() -> None:
     o = ImageCms.getOpenProfile(SRGB)
     p = o.profile
 
-    def assert_truncated_tuple_equal(tup1, tup2, digits: int = 10) -> None:
+    def assert_truncated_tuple_equal(
+        tup1: tuple[tuple[float, float, float], ...] | tuple[float],
+        tup2: (
+            tuple[tuple[tuple[float, float, float], ...], ...]
+            | tuple[tuple[float, float, float], ...]
+            | tuple[float]
+        ),
```

Comment:
Probably okay to make use of `tuple[Any]` or even `Any` here and in `truncate_tuple` to simplify things, but I don't mind too much :)

Reply:
Ok, I've pushed a commit with `tuple[Any, ...]`